dockerfiles/theia-endpoint-runtime/etc/entrypoint.sh

```diff
@@ -47,12 +47,11 @@ set -e
 
 # setup handlers
 # on callback, kill the last background process, which is `tail -f /dev/null` and execute the specified handler
-trap 'responsible_shutdown' SIGHUP SIGTERM SIGINT
-
-cd ${HOME}
+trap 'responsible_shutdown' HUP TERM INT
```

Comment:
is there a reason to change the traps ?

Reply:
Yes, of course, SIGHUP SIGTERM SIGINT doesn't work on fedora.